webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/AnnotationPage.java

```diff
@@ -295,7 +297,7 @@ private void createAnnotationEditor(IPartialPageRequestHandler aTarget)
 
     private SidebarPanel createLeftSidebar()
     {
-        SidebarPanel leftSidebar = new SidebarPanel("leftSidebar", getModel(), detailEditor, () -> 
+        leftSidebar = new SidebarPanel("leftSidebar", getModel(), detailEditor, () -> 
```

Comment:
Why do you assign `leftSidebar` here and in line 209?

Reply:
Thx, good catch, just being too hasty.